app/models/service_template_transformation_plan.rb

```diff
@@ -65,46 +66,23 @@ def self.create_catalog_item(options, _auth_user = nil)
     end
   end
 
-  private
+  def update_catalog_item(options, _auth_user = nil)
+    raise _("Editing a plan in progress is prohibited") if %w(active pending).include?(miq_requests.last.try(:request_state))
```

Comment:
Without proper sorting `miq_requests.last` does not guarantee to be the latest request.

Reply:
Addressed this in the last update